### PR TITLE
Add index

### PR DESCRIPTION
**Pull request by loriculberson**
New index file shows the new layout

### CODE REVIEW

**Review by laxsonii — CHANGES_REQUESTED** [review]
nice

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h2>Hi</h2>
+</body>
+</html>
```
Comment (laxsonii) [inline index.html:11]: change title

File: index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
```
Comment (laxsonii) [inline index.html:7]: add a new div

File: index.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+<body>
+    <h2>Hi</h2>
```
Comment (nducklow) [inline index.html:9]: This should say Dunkin

**Review by nducklow — CHANGES_REQUESTED** [review]
Looks great!

**Review by laxsonii — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (loriculberson) [reply index.html]: Fixed!

File: index.html (hunk 1 above)
Comment (loriculberson) [reply index.html:11]: changed title

**Review by laxsonii — APPROVED** [review]
thank you

**Review by nducklow — APPROVED** [review]

**Pull request merged**